pkg/helm/handlers/handler_test.go

```diff
@@ -290,7 +330,202 @@ func TestHelmHandlers_HandleGetChart(t *testing.T) {
 			if response.Body.String() != tt.expectedResponse {
 				t.Errorf("response body not matching expected is %s and received is %s", tt.expectedResponse, response.Body.String())
 			}
+		})
+	}
+}
+
+func TestHelmHandlers_HandleGetReleaseHistory(t *testing.T) {
+	tests := []struct {
+		name                string
+		expectedResponse    string
+		history             []*release.Release
+		expectedContentType string
+		error
+		httpStatusCode int
+	}{
+		{
+			name:                "Invalid chart release history test",
```

Comment:
please mention here also what is the expected outcome.

Reply:
Done